Task_TheGridProblem/index.css

```diff
@@ -1 +1,79 @@
-/* Start coding below for GRID PROBLEM TASK */
+.container{
+  display:grid;
+  grid-template-rows: 200px 200px 200px;
+  grid-template-columns: 200px 200px 200px;
+  background-color: #a2d2ff;
+  padding: 30px;
+  gap:3px;
```

Comment:
The 'gap' property should have a unit value specified. For example, '3px' instead of '3'.